### ISSUE TITLE
Context menu appears to be broken on release mode only + window focus issue

### ISSUE THREAD

**Issue by widavies** [widavies/WinJump#16]

**Comment by blazej-czapp** [comment]
Hey, I'm enjoying the app, but I'm probably experiencing this issue. In my case the task bar is set to auto-hide. Switching to another desktop focuses some "random" window on some other desktop, causing the task bar to show and obscure the view. I then have to click on several windows on several desktop to figure out which one it is and to remove the focus (just clicking on the window I want doesn't do it). Otherwise very happy :)

**Comment by widavies** [comment]
Right, I think that's probably related to [this issue](https://github.com/MScholtes/VirtualDesktop/issues/57#issuecomment-1518601787) that I'm currently working on. Hope to have a fix up soon for both this issue and that one!

**Comment by widavies** [comment]
@blazej-czapp Latest release should have the context menu fix, still looking into the window focus issue.
